TCAT/Controllers/ServiceAlertsViewController.swift

```diff
@@ -121,24 +121,22 @@ extension ServiceAlertsViewController: UITableViewDataSource {
     }
     
     func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
-        if let alerts = alerts[priorities[section]] {
-            return alerts.count
-        } else {
-            return 0
-        }
+        return alerts[priorities[section]]?.count ?? 0
     }
     
     func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
-        let cell = tableView.dequeueReusableCell(withIdentifier: ServiceAlertTableViewCell.identifier, for: indexPath) as? ServiceAlertTableViewCell
+        
+        var cell : ServiceAlertTableViewCell!
```

Comment:
@drewsdunne @AAAstorga Do we have a good standard way to do this? I mentioned before, I never like using `!`

Reply:
Could guard let? And if it fails, we return the base UITableViewCell (so the program doesn't crash for the user) in addition to showing some sort of error message (either alert or just print some error message so we can debug on our side)

Reply:
I like that!